providers/provider-protocol-reference.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Provider Protocol Reference
+category: 63d7e8bdbf7b4b0e0745e823
+slug: provider-protocol-reference
+---
+
+# Provider Protocol Reference (v2)
+
+All functions decribed here take msgpack-serialized payloads as arguments and
+return values. The key/values to use for these payloads are exactly the ones you
+can find in `fiberplane::models` crate documentation (NOTE: put the docs.rs link
+once fiberplane is open-sourced and published).
+
+## Imported functions
+
+These functions are usable as long as you import `fiberplane_pdk::prelude::*` in
+your library code. These functions exist to allow you to use OS-level features by
+asking the runtime for information.
+
+For Fiberplane Daemon, these functions are implemented in [Provider Runtime
+crate](https://github.com/fiberplane/fiberplane-rs/blob/main/fiberplane-provider-protocol/fiberplane-provider-runtime/src/spec/mod.rs),
+should you want to inspect the source.
+
+### `make_http_request`
+
+#### Signature
+
+```rust
+pub async fn make_http_request(request: HttpRequest) -> Result<HttpResponse, HttpRequestError>;
+```
+
+#### Purpose
+
+This is the most useful imported method, it allows the providers to make HTTP queries to
```

Comment:
A bit nitpicky, but a method is a function that is bound to an instance of something. These are just plain functions :)